docs/generated/packages/expo/executors/start.json

```diff
@@ -24,7 +24,8 @@
         "type": "number",
         "description": "Port to start the native Metro bundler on (does not apply to web or tunnel)",
         "default": 19000,
-        "alias": "p"
+        "alias": "p",
+        "x-priority": "important"
```

Comment:
Check out the guidelines for x-priority flags in asana called `Establish guidelines for x-priority`. 
We decided that `--port` shouldn't be important.

Reply:
removed